shared/js/background/storage/config.es6.js

```diff
@@ -0,0 +1,74 @@
+// Babel includes for async/await
+import 'regenerator-runtime/runtime'
+
+const load = require('./../load.es6')
+const constants = require('../../../data/constants')
+const settings = require('./../settings.es6')
+
+/**
+ *  Manage local cookie exclusion data.
+ *  Data is updated regularly with the latest
+ *  cookie exclusions, and stored here for fast retrieval and usage
+ *  when checking which cookies to block.
+ **/
+class ConfigStorage {
+    constructor () {
+        this.config = {}
```

Comment:
We could use a `Proxy` here to provide a default structure to `config.features` (like a `defaultdict` in Python) when the config is not loaded, or if an attribute is missing from the config. This could guarantee safe access to `config.feature.myFeature.state` at all times.